src/radical/saga/adaptors/srm/srmfile.py

```diff
@@ -84,7 +84,7 @@ class Adaptor(rsab.Base):
     def __init__(self) :
 
         rsab.Base.__init__(self, _ADAPTOR_INFO, _ADAPTOR_OPTIONS)
-        self.pty_url = self._cfg['pty_url']
+        self.pty_url = self._cfg.get('pty_url', 'fork://localhost/')
```

Comment:
Is there a reason you want the pty_url to default to localhost?

Reply:
because that is the documented default value - see line 27.